pkg/apis/kubeflow/v2beta1/constants.go

```diff
@@ -14,15 +14,32 @@
 
 package v2beta1
 
-import common "github.com/kubeflow/common/pkg/apis/common/v1"
-
 const (
 	// EnvKubeflowNamespace is ENV for kubeflow namespace specified by user.
 	EnvKubeflowNamespace = "KUBEFLOW_NAMESPACE"
 	// DefaultRestartPolicy is default RestartPolicy for ReplicaSpec.
-	DefaultRestartPolicy = common.RestartPolicyNever
+	DefaultRestartPolicy = RestartPolicyNever
 	// DefaultLauncherRestartPolicy is default RestartPolicy for Launcher Job.
-	DefaultLauncherRestartPolicy = common.RestartPolicyOnFailure
+	DefaultLauncherRestartPolicy = RestartPolicyOnFailure
 	// OperatorName is the name of the operator used as value to the label common.OperatorLabelName
 	OperatorName = "mpi-operator"
 )
+
+// merge from common.v1
```

Comment:
Can you leave a link to the common repo here?

Reply:
done.